<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findDialog</class>
 <widget class="QDialog" name="findDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <widget class="QGroupBox" name="directionGroupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Direction</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>18</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="upRadioButton">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="downRadioButton">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="caseCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Case</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Find what:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>95</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="findNextButton">
      <property name="text">
       <string>Find Next</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>caseCheckBox</tabstop>
  <tabstop>upRadioButton</tabstop>
  <tabstop>downRadioButton</tabstop>
  <tabstop>findNextButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>findDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findNextButton</sender>
   <signal>clicked()</signal>
   <receiver>findDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
